guest-os-info.md

```diff
@@ -0,0 +1,34 @@
+## Guest Operating System Information
+
+Guest operating system identity for the Virtual Machine will be provided by the label ``kubevirt.io/os-short-id`` :
+
+```
+metadata:
+  name: myvm
+  labels:
+    kubevirt.io/os-short-id: win2k12r2
+```
+
+The ``kubevirt.io/os-short-id`` label is based on the short OS identifier from [libosinfo](https://libosinfo.org/)
```

Comment:
What is the rationale for staying with that clunky name instaed of `kubevirt.io/os`?

Reply:
On one hand it is a clunky name, on the other hand it keeps a logical connection to the origin of the values the label can have  - an OS presentation in short id . Using os-short-id might also prevent confusion with general os type (linux/windows) for example.

Reply:
I would then argue that I would either like to see one of the following:

* `kubevirt.io/os` may contain things like `win2k12r2`, `fedora27` or a family like `windows`.
* `kubevirt.io/os` and `kubevirt.io/os-family` to keep them separate and short, however why would you apply both or contradicting values? If I know it is fedora, I can apply fedora. If I just know it is `linux`, I just apply `linux`.

Especially since we don't plan to introduce `long-id`, it is strange to have it. If you mean by `origin` the connection to libosinfo, that is not really interesting for the end-user.

Reply:
ok, so lets have the ``kubevirt.io/os`` label instead of ``kubevirt.io/os-short-id``. 
for non end-users it would be helpful to know what it is based on - but for that i have added a developers doc. 
and if we will require extra labels such as os-family then we will add them later on. 
(main idea with short-id is not to overload labeles with os info and use it to query libosinfo database for additional info - but as you mentioned, this is not for end users)

Reply:
BTW, at the moment we use `annotations` to get `os-family`. From client perspective there is value in having it.

Reply:
@pkliczewski so this is for visualisation and orientation? Sounds good.

Reply:
@rmohr correct